kubernetes/samples/scripts/create-weblogic-domain/manually-create-domain/domain.yaml

```diff
@@ -65,7 +65,10 @@ spec:
     - name: JAVA_OPTIONS
       value: "-Dweblogic.StdoutDebugEnabled=false"
     - name: USER_MEM_ARGS
-      value: "-Xms64m -Xmx256m "
+      value: "-Xms256m -Xmx512m "
```

Comment:
It looks like USER_MEM_ARGS here should have the urandom.  A long-standing error - can you please add it?

Reply:
Added urandom in USER_MEM_ARGS.